pages/create-issue.tsx

```diff
@@ -61,45 +114,51 @@ export default function PageCreateIssue() {
         <form onSubmit={createIssue}>
           <div className="container">
             <div className="row justify-content-center">
-              <div className="col-md-6">
+              <div className="col-md-10">
                 <div className="content-wrapper mt-up mb-5">
-                  <h4 className="h4 mr-2">Lock</h4>
+                  <h3 className="h3 mr-2 mb-4">Details</h3>
                   <div className="form-group mb-4">
-                    <label className="p-small trans mb-2">Issue title</label>
+                    <label className="p-small mb-2">Issue title</label>
                     <input type="text"
                       className="form-control" placeholder="Your issue title"
                       value={issueTitle}
                       onChange={e => setIssueTitle(e.target.value)}
                       />
+                    <p className="p-small trans my-2">Tip: Try to be as much descriptive as possible</p>
                   </div>
-                  <div className="form-group mb-4">
-                    <label className="p-small trans mb-2">Set $BEPRO ammout</label>
-                    <input min="0" type="number" className="form-control" placeholder="0"
-                      value={issueAmount}
-                      onChange={e => setIssueAmount(parseInt(e.target.value))}/>
+                  <div className="form-group col-md-4 mb-4">
+                    <label className="p-small mb-2">Set $BEPRO value</label>
```

Comment:
If you take a look in general for all inputs, this label could be within `InputNumber` too! So, it could render the label through `[label]`.

If other props has a value per default, you can set it directly within, such `thousandSeparator` if so.

Reply:
Done. -> 741c2a7. does this solve?

Reply:
It does! 😍 I just would remove this `[symbol]` default value, because in the future we may use this `InputNumber` for other purposes besides entry balance value. Don't you agree? So:

```jsx
  symbol = "",
```
```jsx
  {symbol && <span>{symbol}</span>}
```